linux_docker_resources/entry_point.sh

```diff
@@ -20,11 +20,11 @@ echo "Enabling multicast..."
 ifconfig eth0 multicast
 echo "done."
 
-case "${CI_ARGS}" in
-  *rmw_connext_cpp*)
-    echo "NOT installing Connext."
-    ;;
-  *)
+# extract all ignored rmws
+# extract args between --ignore-rmw until the first appearance of '-'
+IGNORE_CONNEXT=`echo ${CI_ARGS} | sed -e 's/.*ignore-rmw \(.*\)--.*/\1/' | sed -e 's/-.*//' | grep rmw_connext_cpp`
```

Comment:
I think the regex needs tuning - I'm not sure it will work right if `--ignore-rmw foo` is the last argument.

Maybe something like this? `s/.*--ignore-rmw \([^-]*\).*/\1/`

Reply:
makes a lot of sense to me. Updated in https://github.com/ros2/ci/pull/345/commits/f430664735dc4c3e290de57d11691d5cac39a549